radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -299,30 +314,49 @@ impl FeeTable {
                 node_id: _,
                 partition_num: _,
                 substate_key: _,
-            } => 632, // todo: determine correct value
-            CostingEntry::LockSubstateFirstTime => 100, // todo: determine correct value
+            } => 100, // todo: determine correct value
+            CostingEntry::ScanSubstate => 16, // todo: determine correct value
+            CostingEntry::SetSubstate => 16,  // todo: determine correct value
+            CostingEntry::TakeSubstate => 16, // todo: determine correct value
             CostingEntry::ReadSubstate { size: _ } => 174,
             CostingEntry::WriteSubstate { size: _ } => 126,
+
+            // following variants are used in storage usage part only
+            CostingEntry::SubstateReadFromDb { size: _ } => 0,
+            CostingEntry::SubstateReadFromTrack { size: _ } => 0,
+            CostingEntry::SubstateWriteToTrack { size: _ } => 0,
+            CostingEntry::SubstateRewriteToTrack {
+                size_old: _,
+                size_new: _,
+            } => 0,
         }) as u64
-            * COSTING_COEFFICENT
-            >> (COSTING_COEFFICENT_DIV_BITS + COSTING_COEFFICENT_DIV_BITS_ADDON)) as u32
+            * COSTING_COEFFICENT_CPU
+            >> (COSTING_COEFFICENT_CPU_DIV_BITS + COSTING_COEFFICENT_CPU_DIV_BITS_ADDON))
+            as u32
     }
 
-    fn kernel_api_cost_from_memory_usage(&self, entry: &CostingEntry) -> u32 {
-        match entry {
-            CostingEntry::CreateNode { size, node_id: _ } => 100 * size,
-            CostingEntry::DropNode { size } => 100 * size,
+    fn kernel_api_cost_storage_usage(&self, entry: &CostingEntry) -> u32 {
+        ((match entry {
             CostingEntry::Invoke {
                 input_size,
                 actor: _,
             } => 10 * input_size,
-            CostingEntry::ReadSubstate { size } => 10 * size,
+            CostingEntry::ReadSubstate { size } => 100 * size,
             CostingEntry::WriteSubstate { size } => 1000 * size,
+            CostingEntry::SubstateReadFromDb { size } => 100 * size, // todo: determine correct value
```

Comment:
We should move `ReadSubstate` and `WriteSubstate` to computation cost, as the compuation cost is "linear" to size because of reading/copying/transforming memory chunks?